task/create.html

```diff
@@ -17,28 +17,39 @@
             class="input"
             type="text"
             name="title"
-            placeholder="Title"
+            placeholder="eg: Testing setup for dashboard"
             id="title"
           />
         </div>
         <div class="inputBox">
-          <label for="purpose">Purpose:</label>
+          <div class="checkBox">
+            <label for="isNoteworthy">Is noteworthy:</label>
+            <span
+              ><input type="checkbox" id="isNoteworthy" name="isNoteworthy"
```

Comment:
Can you please format your code using prettier(need to run npm install) and run npm fix

Reply:
I did run npm fix, and all checks have passed

Reply:
Yes that's right, but linting does not format your code but it checks for a few rules 
So please format your code and run **npm fix** as well
Thank you :smiley:

Reply:
When I run **npm run fix** nothing changed

Reply:
Got it :+1: 
We have a linting issue will fix that